src/dwarf.cc

```diff
@@ -591,6 +592,25 @@ struct DwoFilePointer {
   std::string dwo_name;
 };
 
+static std::string ConstructDwoPath(const DwoFilePointer& dwo_info) {
+  if (dwo_info.dwo_name.empty()) {
+    return "";
+  }
+
+  std::filesystem::path dwo_path(dwo_info.dwo_name);
+  if (dwo_path.is_absolute()) {
+    // Absolute path
+    return dwo_info.dwo_name;
+  } else if (!dwo_info.comp_dir.empty()) {
```

Comment:
Very minor nit, I don't think we need to check if the `comp_dir` is empty, path should do the right thing.